HappyCamps/src/app/Models/event.ts

```diff
@@ -0,0 +1,12 @@
+import { User } from "./user";
+
+export interface Event {
+    id: number;
```

Comment:
Why is the id a number and not a Guid?

Reply:
Why would be a Guid?

Reply:
because an number id can get stack overflow and it's less secure